modules/biobank/php/container.class.inc

```diff
@@ -0,0 +1,350 @@
+    {
+        $this->barcode = $barcode;
+    }
+
+    /**
+     * Gets the container's barcode
+     *
+     * @return string
+     */
+    public function getBarcode() : string
+    {
+        return $this->barcode;
+    }
+
+    /**
+     * Sets the ID of the container type
+     *
+     * @param int $typeId ID of the container type
+     *
+     * @return void
+     */
+    public function setTypeId(int $typeId) : void
+    {
+        $this->typeId = $typeId;
+    }
+
+    /**
+     * Gets the ID of the container type
+     *
+     * @return int
+     */
+    public function getTypeId() : int
+    {
+        return $this->typeId;
+    }
+
+    /**
+     * Sets the ID of the container's dimensions
+     *
+     * @param int $dimensionId ID of the container's dimensions
+     *
+     * @return void
+     */
+    public function setDimensionId(int $dimensionId) : void
+    {
+        $this->dimensionId = $dimensionId;
+    }
+
+    /**
+     * Gets the ID of the container's dimensions
+     *
+     * @return int
+     */
+    public function getDimensionId() : int
+    {
+        return $this->dimensionId;
+    }
+
+    /**
+     * Sets the container's temperature in Celsius.
+     *
+     * @param float $temperature the container's temperature.
+     *
+     * @return void
+     */
+    public function setTemperature(float $temperature) : void
+    {
+        $this->temperature = $temperature;
+    }
+
+    /**
+     * Gets the container's temperature in Celsius.
+     *
+     * @return float
+     */
+    public function getTemperature() : float
+    {
+        return $this->temperature;
+    }
+
+    /**
+     * Sets the ID of the container's current status
+     *
+     * @param int $statusId the ID of the container's current status
+     *
+     * @return void
+     */
+    public function setStatusId(int $statusId) : void
+    {
+        $this->statusId = $statusId;
+    }
+
+    /**
+     * Gets the ID of the container's current status
+     *
+     * @return int
+     */
+    public function getStatusId() : int
+    {
+        return $this->statusId;
+    }
+
+    /**
+     * Sets the ID of the container's origin
+     *
+     * @param int $originId the ID of the container's origin
+     *
+     * @return void
+     */
+    public function setOriginId(int $originId) : void
+    {
+        $this->originId = $originId;
+    }
+
+    /**
+     * Gets the ID of the container's origin
+     *
+     * @return int
+     */
+    public function getOriginId() : int
+    {
+        return $this->originId;
+    }
+
+    /**
+     * Sets the ID of the container's current centerId
+     *
+     * @param int $centerId the ID of the container's current centerId
+     *
+     * @return none
+     */
+    public function setCenterId(int $centerId) : void
+    {
+        $this->centerId = $centerId;
+    }
+
+    /**
+     * Gets the ID of the container's current centerId
+     *
+     * @return int
+     */
+    public function getCenterID() : int
```

Comment:
@HenriRabalais I would prefer functions like this one `somethingID` with an uppercase convention rather than `somethingId`. Like Rida said, it would be nice to stay consistent, so all `ID` or all `Id`.

Reply:
I think this is the only occasion where there's an uppercase `ID`, so I must have made a mistake. I'll switch it to `getCenterId()`.

Reply:
Sounds good, it is consistent now